lib/kafka_ex/protocol.ex

```diff
@@ -37,6 +38,10 @@ defmodule KafkaEx.Protocol do
     @consumer_metadata_request
   end
 
+  defp api_key(:join_group) do
+    11
```

Comment:
Funny. I have numeric constants in all the other calls. I must be messing up with Git somewhere :-). Will fix it.